game-core/src/main/java/games/strategy/triplea/delegate/battle/MustFightBattle.java

```diff
@@ -1494,45 +1496,53 @@ public void execute(final ExecutionStack stack, final IDelegateBridge bridge) {
       }
     };
 
-    final ReturnFire returnFireAgainstAttackingSubs =
-        SubsChecks.returnFireAgainstAttackingSubs(attackingUnits, defendingUnits, gameData);
-    final ReturnFire returnFireAgainstDefendingSubs =
-        SubsChecks.returnFireAgainstDefendingSubs(attackingUnits, defendingUnits, gameData);
-    if (SubsChecks.defenderSubsFireFirst(attackingUnits, defendingUnits, gameData)) {
+    if (firstStrikeStepOrders.contains(FirstStrikeStepOrder.FIRST_STRIKE_DEFENDER_FIRST_NONE)) {
```

Comment:
naming nit:
- FIRST_STRIKE prefix is perhaps redundant on an enum called `FirstStrikeStepOrder`. WDYT of this being "FirstStrikeStepOrder.DEFENDER_FIRST_NONE`
- `DEFENDER_FIRST_NONE` does not initially make as much sense as it could. Can you suggest a better name perhaps? Maybe `DEFENDERS_DO_NOT_RETURN_FIRE`. Though, I can't help but feel that is a weird step order, it seems like a 'null' step and should just not be a step at all.. Hence, it's a bit difficult to grok what is intended by this step order from the name.